docs/contributing/999-rfcs/2024-06-26-wing-docs-v2.md

```diff
@@ -0,0 +1,161 @@
+---
+title: RFC Template
+description: Template for creating a new RFC
+---
+
+# Wing documentation v2
+
+- **Author(s)**: @boyney213
+- **Submission Date**: 2024-06-26
+- **Stage**: Proposal
+
+## Overview
+This RFC outlines changes and features for the [wing documentation](https://www.winglang.io/docs/) v2.
+
+Docs v2 aims to simplify the initial onboarding of wing, organise content and define the information architecture of the documentation website.
+
+The vision for Wing's documentation is to elevate the experience for all types of learners—visual, auditory, and hands-on. Getting started with wing should take minutes and the documentation needs to be accessible (easy to find), clean (reduce noise) and automated where possible (e.g api docs, compiling example code in the docs against wing, winglibs automation). 
+
+This document contains areas of refactoring and new ideas for wing documentation with various implementation phases.
+
+## Wing Personas
+
+It’s important to understand and agree the target audience of wing as the documentation will serve multiple users. These personas can be split into three categories, **full-stack developers**, **platform engineers** and **contributors**.
+
+**Full stack developers -** Developers building applications using wing. These developers may not understand the basics of the cloud so terms may need to be explained to them (i.e Queues, Functions, Buckets) and may be engineers that are in Kubernetes shops. 
+
+These developers are consumers of wing libs and the wing standard library. They are part of stream aligned teams focused on adding business value, not necessary worried about infrastructure and operate with abstractions into the cloud.
+
+**Platform engineers** - Engineers that are focused on building platforms for other internal developers, focusing mainly on developing custom platforms or wing libs for others to consume. 
+
+Platform engineers understand the basics of the cloud (Queues, Functions, Buckets). These engineers are builders of wing libs and platforms. They are focused on agility within the business, building tools and platforms for engineers to move faster, with clear abstractions into their custom platforms and the cloud.
+
+**Contributor** - Open source contributors to wing. They understand the basics of the cloud and are looking for ways to contribute ideas, code and resources to wing.
```

Comment:
nit:
Not all contributors may have knowledge of the cloud, maybe they are just enthusiastic language hackers who want to work on the rust compiler.

Reply:
Fair point, will amend